liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -108,6 +108,12 @@
     @Parameter(property = "debug.port", defaultValue = "7777")
     private int libertyDebugPort;
 
+    /**
+     * Time in seconds to wait before processing Java changes and deletions.
+     */
+    @Parameter(property = "compile.wait", defaultValue = "0.5")
```

Comment:
I believe we should be using camel case for the parameter names, as is the Maven convention: https://stackoverflow.com/questions/30297777/convention-for-maven-properties-dot-case-or-camel-case

Reply:
This would also apply to the debug port

Reply:
Changed to camel case for compileWait and debugPort